compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUInterfaces.td

```diff
@@ -104,6 +104,69 @@ def IREEGPU_MmaInterfaceAttr : AttrInterface<"MmaInterfaceAttr"> {
         return failure();
       }]
     >,
+    InterfaceMethod<
+      /*desc=*/[{
+        Constructs the offsets/sizes/strides for extracting the per-thread
+        slice of the LHS.
+      }],
+      /*retTy=*/"::mlir::LogicalResult",
+      /*methodName=*/"populateLhsOffsetsSizesStrides",
```

Comment:
Should we make lhs/rhs/acc a function argument so that we only need one interface method?

Reply:
nice, this ended up being a bit less code duplication.